sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -336,8 +336,11 @@ caseFuncAlternative
     ;
 
 aggregateFunction
-    : functionName=aggregationFunctionName LR_BRACKET functionArg RR_BRACKET #regularAggregateFunctionCall
-    | COUNT LR_BRACKET STAR RR_BRACKET                                       #countStarFunctionCall
+    : functionName=aggregationFunctionName LR_BRACKET functionArg RR_BRACKET
+                                                                    #regularAggregateFunctionCall
+    | functionName=aggregationFunctionName LR_BRACKET DISTINCT functionArg RR_BRACKET
```

Comment:
Added the test cases in `WindowIT.java` because the comparison test failed, the schema does not match with H2 and the SQLite does not support distinct keyword in window function.

Reply:
Got it. You can move it to `WindowFunctionIT`. Ignore this comment if that takes time. Thanks!

Reply:
@dai-chen Sorry it was typo, I have added the cases in `WindowFunctionIT`.